extensions/php/snippets/php.snippets.json

```diff
@@ -254,8 +254,8 @@
 		"body": [
 			"try {",
 			"\t${1://code...}",
-			"} catch (\\Exception \\$$e) {",
-			"\t${2://throw $$e;}",
+			"} catch (${2:\\Throwable} ${3:$$t}) {",
```

Comment:
Is this supposed to be `$th`?

Reply:
Oops! Changed https://github.com/Microsoft/vscode/pull/60012/commits/1de8905bf5c0e26170aff0d7b3332de9f9f5ffe3